README.md

```diff
@@ -47,9 +53,13 @@ The plugin requires an active Parse.ly account. Parse.ly gives creators, markete
 4. Visit the Plugins page from your WordPress dashboard and look for the newly installed Parse.ly plugin.
 5. Click _Activate_ to activate the plugin on your site.
 
+Note that this method is the recommended one for installing old versions of the plugin. Those can be downloaded from [WordPress.org](https://wordpress.org/plugins/wp-parsely/advanced/) or GitHub Releases page.
```

Comment:
"GitHub Releases page" could also be a link. Also maybe add a "the" before?

Reply:
Not adding the link because it's linked just above already.